### PR TITLE
feat!: release v2

### PR DESCRIPTION
**Pull request by mshaaban0**
## Summary

Release beta v2 to `contentful-cli@latest`

## Release Notes

* Typescript support
* New `init` command to get started with a new or existing Contentful space
* Scaffold for the upcoming `merge` commands. [WIP]
* Auto complete on space selection
* Removed `guide` command
* Removed `snake` command
* Removed `Linux` `Mac OSX` standalone packages

## Todos

- [x] Sanity check
- [x] Release notes are ready

### CODE REVIEW

**Review by crissto — APPROVED** [review]
🚀

**Comment by marcolink** [comment]
Does the `chore!:`  exclamation mark has any impact? 🤔

**Review by denkristoffer — COMMENTED** [review]
Bit late with this but I'm wondering why bother with Babel over tsc for compilation?

File: tsconfig.json [hunk 1]
```diff
@@ -1,16 +1,104 @@
...
-    "allowJs": true,
-    "skipLibCheck": true,
-    "strict": true,
-    "module": "CommonJS",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "outDir": "output"
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+
+    /* Projects */
```
Comment (marcolink) [inline tsconfig.json:5]: Should we remove al the comments here?

**Comment by mshaaban0** [comment]
> Does the `chore!:` exclamation mark has any impact?

Yes, it means a breaking change for semantic releases https://www.conventionalcommits.org/en/v1.0.0/#summary

File: .circleci/config.yml [hunk 2]
```diff
@@ -101,17 +102,19 @@ workflows:
     jobs:
       - unit
       - integration
-      - linux
-      - osx
+      # - linux
+      # - osx
+      # - audit
```
[the remark this answers is not in the document]
Comment (mshaaban0) [reply .circleci/config.yml:107]: We only need `audit` thanks. resolved #1823

File: package.json [hunk 3]
```diff
@@ -13,39 +13,45 @@
   "author": "Contentful <opensource@contentful.com>",
   "license": "MIT",
   "scripts": {
-    "tsc": "tsc",
     "audit": "resolve-audit",
-    "build:standalone": "npm run tsc && pkg --targets node16-macos-x64,node16-linux-x64,node16-win-x64 --out-dir build .",
+    "build:standalone": "npm run tsc && pkg --targets node12-macos-x64,node12-linux-x64,node12-win-x64 --out-dir build .",
```
[the remark this answers is not in the document]
Comment (mshaaban0) [reply package.json:17]: Removed in #1823

File: package.json [hunk 4]
```diff
@@ -55,43 +61,47 @@
     "eslint-plugin-prettier": "^4.0.0",
     "http-proxy": "^1.17.0",
     "husky": "^4.3.8",
-    "jest": "^26.6.3",
+    "jest": "^28.1.3",
     "jszip": "^3.7.1",
     "lint-staged": "^13.0.0",
     "nixt": "^0.5.1",
     "nyc": "^15.0.0",
+    "parcel": "^2.6.2",
```
[the remark this answers is not in the document]
Comment (mshaaban0) [reply package.json:69]: Thanks, removed

File: tsconfig.json (hunk 1 above)
Comment (mshaaban0) [reply tsconfig.json:5]: done

**Comment by mshaaban0** [comment]
> Bit late with this but I'm wondering why bother with Babel over tsc for compilation?

@denkristoffer for compilation we only use `tsc`, babel is only for writing jest in typescript

**Comment by denkristoffer** [comment]
@mshaaban0 I see, makes sense. Damn jest.

**Review by denkristoffer — APPROVED** [review]
Looks good to me then.

**Pull request merged**